supplementary_style_guide/glossary_terms_conventions/general_conventions/v.adoc

```diff
@@ -37,7 +37,7 @@ Use lowercase "v" and uppercase "CPU".
 [discrete]
 [[verb]]
 ==== image:images/yes.png[yes] verb (noun)
-*Description*: In Red Hat OpenShift, a verb is a get, list, create, or update operation.
+*Description*: In Red Hat OpenShift, a _verb_ is a get, list, create, or update operation.
```

Comment:
"verb" as it refers to language use?

Reply:
I don't think it does, it's talking about the technical concept of it, which is a "get", "list", "create", or "update" operation. I'm thinking that it would help if these items were in backticks probably. I can update to add those.